changelogs/unreleased/7676-metrics-retention.yml

```diff
@@ -0,0 +1,6 @@
+description: Reduce metrics retention time to two weeks
+issue-nr: 7676
+change-type: minor
+destination-branches: [master, iso7, iso6]
```

Comment:
I'm not quite sure IF we should also add this to iso6

Reply:
Probably not but we can do it as a safety measure